docs/source/cpp/compute.rst

```diff
@@ -1549,6 +1549,17 @@ These functions select and return a subset of their input.
 * \(4) For each element *i* in input 2 (the indices), the *i*'th element
   in input 1 (the values) is appended to the output.
 
+Containment tests
+~~~~~~~~~~~~~~~~~
+
+This functions return the indices at which elements match a predicate
+
++-----------------------+-------+-----------------------------------+----------------+---------------------------------+-------+
+| Function name         | Arity | Input types                       | Output type    | Options class                   | Notes |
```

Comment:
No need to add "Options class" and "Notes" columns if they remain empty, IMHO.

Reply:
I thought about this initially, but I think that keeping those two columns will make life easier for additional compute functions that might have notes or options. Given that indentation in rst tables makes easy to mess them up, already preparing the columns will make life easier for future additions.